mrpt_graphslam_2d/CMakeLists.txt

```diff
@@ -18,3 +18,47 @@ if (BUILD_TESTING)
     ${test_output_directory}/${PROJECT_TEST_NAME})
 endif()
 
+set(PROJECT_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+
+#V-REP Config
+################################################################################
+
+if (EXISTS "$ENV{VREP}")
+  message(STATUS "Found V-REP")
+  set (V-REP_ROOT "${PROJECT_ROOT_DIR}/../" CACHE "V-REP root dir" STRING)
+
+
+  set (V-REP_PATH "${PROJECT_ROOT_DIR}/../V-REP/programming/" CACHE "The V-REP build path" STRING)
+
+  set (SOURCES remoteApi/bubbleRob_test.cpp
+    ${V-REP_ROOT}/V-REP/programming/remoteApi/extApi.c
```

Comment:
Minor, but do order them nicely... `.h` first, `.c` afterwards

Reply:
Correct if I'm wrong but from what I also see below, I assume that your aim is to make all the v-rep stuff into a single library, which you later link to the executable that actually uses them. If so, then you should remove "remoteApi/bubleRob_test.cpp" from this list and add it only to the corresponding "make_executable" command.

Reply:
Yes. This is what my aim is. Thanks this way is cleaner and easier to understand